config.ini

```diff
@@ -0,0 +1,92 @@
+# EOL and security-fixes are not automatically built, no need to remove them
+# from the list, this way we can still easily rebuild them manually as needed.
+#
+# Please keep the list in reverse-order for ease of editing.
+
+[3.12]
+branch = origin/main
+status = in development
+
+[3.11]
+branch = origin/3.11
+status = stable
+
+[3.10]
+branch = origin/3.10
+status = stable
+
+[3.9]
+branch = origin/3.9
+status = security-fixes
+
+[3.8]
+branch = origin/3.8
+status = security-fixes
+
+[3.7]
+branch = origin/3.7
+status = security-fixes
+
+[3.6]
+tag = 3.6
+status = EOL
+
+[3.5]
+tag = 3.5
+status = EOL
+
+[2.7]
+tag = 2.7
+status = EOL
```

Comment:
Can you use https://github.com/python/devguide/blob/main/include/release-cycle.json instead of duplicating information?

Reply:
I was unaware of this file, I'll gladly use it yes! Let me take a look :)